content/en/user-manual/graphics/shader-chunk-migrations.md

```diff
@@ -41,7 +128,7 @@ The following tables break down the chunk changes by Engine release.
 
 ---
 
-#### Engine v1.57
+#### *Engine v1.57*
```

Comment:
Do we want to bold all titles in this page 😅 ?

Reply:
I removed the italics, and increased the Engine headers as they were almost undetectable.